quantus/metrics/robustness/avg_sensitivity.py

```diff
@@ -307,6 +312,15 @@ def evaluate_batch(
                 **self.perturb_func_kwargs,
             )
 
+            changed_prediction_indexes = (
+                np.argwhere(
```

Comment:
Have you checked that this works with 1. tabular 2. image and 3. 1D data?

Reply:
I have added a test case with image data for `AvgSensitivity`, `Continuity`, `LocalLipschitzEstimate`, `MaxSensitivity`.
I have added a test case with 1D data for `AvgSensitivity`, `LocalLipschitzEstimate`, `MaxSensitivity`. No `Continuity` since it does not support 1D data (all test cases with 1D data expect Error).
However, I did not add test cases for tabular data because I did not find any existing tests for tabular data 🤷

Reply:
here is the issue https://github.com/understandable-machine-intelligence-lab/Quantus/issues/187. 
I will add them over the course of next days in separate PR.